src/BaselineOfMuTalk/BaselineOfMuTalk.class.st

```diff
@@ -15,12 +16,13 @@ BaselineOfMuTalk >> baseline: spec [
 			package: 'MuTalk-TestResources' with: [ spec requires: #( 'MuTalk-Model' ) ];
 			package: 'MuTalk-CI' with: [ spec requires: #( 'MuTalk-Model' ) ];
 			package: 'MuTalk-CI-Tests' with: [ spec requires: #( 'MuTalk-Model' 'MuTalk-CI' ) ];
-			package: 'MuTalk-Tests' with: [ spec requires: #( 'MuTalk-Model' 'MuTalk-TestResources' ) ];
-			package: 'MuTalk-SpecUI' with: [ spec requires: #('MuTalk-Model') ].
+			package: 'MuTalk-Tests' with: [ spec requires: #( 'MuTalk-Model' 'MuTalk-TestResources' 'MuTalk-Utilities' ) ];
+			package: 'MuTalk-SpecUI' with: [ spec requires: #('MuTalk-Model') ];
+			package: 'MuTalk-Utilities' with: [ spec requires: #( 'MuTalk-Model' ) ].
 
```

Comment:
Well now with the new package we sould add 'Mutalk-Utilities-Tests' in this Baseline! :)

Reply:
Done!